### PR TITLE
Complete the gemspec metadata

### PR DESCRIPTION
**Pull request by olleolleolle**
This adds all links from the RubyGems site.

### CODE REVIEW

File: letter_opener.gemspec [hunk 1]
```diff
@@ -18,8 +18,12 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3.4"
   
   if s.respond_to?(:metadata)
-    s.metadata['changelog_uri'] = 'https://github.com/ryanb/letter_opener/blob/master/CHANGELOG.md'
-    s.metadata['source_code_uri'] = 'https://github.com/ryanb/letter_opener/'
-    s.metadata['bug_tracker_uri'] = 'https://github.com/ryanb/letter_opener/issues'
+    s.metadata = {
+      'bug_tracker_uri' = 'https://github.com/ryanb/letter_opener/issues',
```
Comment (nashby) [inline letter_opener.gemspec]: typo :)

Comment (olleolleolle) [reply letter_opener.gemspec]: Thanks so much!

Comment (olleolleolle) [reply letter_opener.gemspec]: Hope  you can squash this, if not, tell me, and I'll squash locally.

Comment (nashby) [reply letter_opener.gemspec]: Please do :)

Comment (olleolleolle) [reply letter_opener.gemspec]: OK, there you are.

File: letter_opener.gemspec [hunk 2]
```diff
@@ -18,8 +18,12 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3.4"
   
   if s.respond_to?(:metadata)
-    s.metadata['changelog_uri'] = 'https://github.com/ryanb/letter_opener/blob/master/CHANGELOG.md'
-    s.metadata['source_code_uri'] = 'https://github.com/ryanb/letter_opener/'
-    s.metadata['bug_tracker_uri'] = 'https://github.com/ryanb/letter_opener/issues'
+    s.metadata = {
+      'bug_tracker_uri' => 'https://github.com/ryanb/letter_opener/issues',
+      'changelog_uri' => 'https://github.com/ryanb/letter_opener/blob/master/CHANGELOG.md',
+      'documentation_uri' => 'http://www.rubydoc.info/gems/letter_opener/'
```
Comment (nashby) [inline letter_opener.gemspec]: oops, missing comma

Comment (olleolleolle) [reply letter_opener.gemspec]: Thanks, it's been a long day.

**Comment by nashby** [comment]
Thanks!

**Pull request merged**